recipes/snorkel-metal/meta.yaml

```diff
@@ -0,0 +1,66 @@
+{% set name = "snorkel-metal" %}
+{% set version = "0.1.0" %}
+{% set file_ext = "tar.gz" %}
+{% set hash_type = "sha256" %}
+{% set hash_value = "a0317439bd7269d9275472a5f4b09b91d907ea4b3be7bc0858e1ec01c55421c4" %}
+
+package:
+  name: '{{ name|lower }}'
+  version: '{{ version }}'
+
+source:
+  fn: '{{ name }}-{{ version }}.{{ file_ext }}'
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.{{ file_ext }}
+  '{{ hash_type }}': '{{ hash_value }}'
+
+build:
```

Comment:
This package looks like it might be a candidate for `noarch: python`:

https://conda-forge.org/docs/meta.html#building-noarch-packages

Please note the comment about not using the `skip` with selector and instead constrain the python version.

Reply:
Great! I've added the noarch setting.